mzLib/Test/TestSpectralSimilarity.cs

```diff
@@ -47,6 +47,8 @@ public void TestAllSpectrumSimilaritiesWithoutMzFilter()
             Assert.That(s.PearsonsCorrelation(), Is.EqualTo(-0.87).Within(0.01));
             Assert.That(s.DotProduct(), Is.EqualTo(0.17).Within(0.01));
 
+            double? test = s.SpectralEntropy();
+
```

Comment:
I didn't see a real test for SpectralEntropy here.  Did I miss something? How about something like "Assert.That(test, Is.EqualTo(??).Within(??))".

Reply:
Good idea Yuling, thank you. Implemented as suggested.